pydasi/dasi/query.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2023 European Centre for Medium-Range Weather Forecasts (ECMWF)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ._dasi_cffi import ffi, ffi_decode, ffi_encode, lib, DASIException
+from .utils import DEBUG, getLogger
+
+logger = getLogger(__name__)
+logger.setLevel(DEBUG)
+
+
+class Query:
+    """
+    Container for the keyword:value pairs that is used for retrieving data.
+    """
+
+    def __init__(self, pair=None):
```

Comment:
Once again, I really don't like this call signature. Please think about what it should look like.